contrib/stack/topsStack/Stack.py

```diff
@@ -333,7 +333,7 @@ def unpackStackReferenceSLC(self, safe_dict):
         configObj.outDir = os.path.join(self.work_dir, 'reference')
         configObj.geom_referenceDir = os.path.join(self.work_dir, 'geom_reference')
         configObj.dem = os.path.join(self.work_dir, configObj.dem)
-        configObj.numProcess = self.numProcess
+        configObj.numProcess = self.numProcess4topo
```

Comment:
Since `self/configObj.numProcess4topo` is used in `topo()`, we should not need this line anymore.